data-model/src/main/avro/RawEventLog.avdl

```diff
@@ -0,0 +1,19 @@
+@namespace("org.hypertrace.core.datamodel")
+protocol LogEventRecordsProtocol {
+  import idl "Attributes.avdl";
+
+  record LogEventRecord {
+    string customer_id;
+
+    bytes span_id;
+    bytes trace_id;
+
+    long time_stamp = 0;
```

Comment:
nit: Can you add a comment on what is this time represents - ephoc, ms/nanos?

Reply:
Please let's use microtime, nano is way too much and millis is not good enough as many spans can last only micros (e.g. 40 microsecs).

Reply:
Using microtime now

Reply:
nanos does feel a bit overkill, but since this is greenfield, why not match the incoming data (since OTLP works in nanos)? I agree that I'm hard-pressed to see the use case - but some user in the future might and there's no difference in storage requirements, so may as well maintain full resolution.

Reply:
Yeah, storing in full resolution makes sense.